.github/CODEOWNERS

```diff
@@ -5,8 +5,8 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
```

Comment:
This also needs to be updated in `.repo-metadata.json` otherwise this file will be overwritten on next generation. Example: https://github.com/googleapis/java-bigtable/blob/master/.repo-metadata.json#L12